frame/scheduler/src/lib.rs

```diff
@@ -187,14 +202,135 @@ decl_error! {
 		/// Reschedule failed because it does not change scheduled time.
 		RescheduleNoChange,
 	}
-}
 
-decl_module! {
-	/// Scheduler module declaration.
-	pub struct Module<T: Config> for enum Call where origin: <T as system::Config>::Origin {
-		type Error = Error<T>;
-		fn deposit_event() = default;
+	#[pallet::genesis_config]
+ 	pub struct GenesisConfig;
+
+ 	#[cfg(feature = "std")]
+ 	impl Default for GenesisConfig {
+ 		fn default() -> Self{
+			 Self
+		 }
```

Comment:
Small whitespace nits:

```suggestion
 		fn default() -> Self {
			Self
		}
```